vscode/src/non-stop/FixupController.ts

```diff
@@ -59,31 +59,61 @@ export class FixupController
+                telemetryService.log('CodyVSCodeExtension:fixup:codeLens:clicked', { op: 'cancel', hasV2Event: true })
+                telemetryRecorder.recordEvent('cody.fixup.codeLens.cancel', 'clicked', {
+                    privateMetadata: { op: 'cancel' },
+                })
                 return this.cancel(id)
             }),
             vscode.commands.registerCommand('cody.fixup.codelens.diff', id => {
-                telemetryService.log('CodyVSCodeExtension:fixup:codeLens:clicked', { op: 'diff' })
+                telemetryService.log('CodyVSCodeExtension:fixup:codeLens:clicked', { op: 'diff', hasV2Event: true })
+                telemetryRecorder.recordEvent('cody.fixup.codeLens.diff', 'clicked', {
+                    privateMetadata: { op: 'diff' },
+                })
                 return this.diff(id)
             }),
             vscode.commands.registerCommand('cody.fixup.codelens.retry', async id => {
-                telemetryService.log('CodyVSCodeExtension:fixup:codeLens:clicked', { op: 'regenerate' })
+                telemetryService.log('CodyVSCodeExtension:fixup:codeLens:clicked', {
+                    op: 'regenerate',
+                    hasV2Event: true,
+                })
+                telemetryRecorder.recordEvent('cody.fixup.codeLens.retry', 'clicked', {
+                    privateMetadata: { op: 'regenerate' },
+                })
                 return this.retry(id)
             }),
             vscode.commands.registerCommand('cody.fixup.codelens.undo', id => {
-                telemetryService.log('CodyVSCodeExtension:fixup:codeLens:clicked', { op: 'undo' })
+                telemetryService.log('CodyVSCodeExtension:fixup:codeLens:clicked', { op: 'undo', hasV2Event: true })
+                telemetryRecorder.recordEvent('cody.fixup.codeLens.undo', 'clicked', {
+                    privateMetadata: { op: 'undo' },
+                })
                 return this.undo(id)
             }),
             vscode.commands.registerCommand('cody.fixup.codelens.accept', id => {
-                telemetryService.log('CodyVSCodeExtension:fixup:codeLens:clicked', { op: 'accept' })
+                telemetryService.log('CodyVSCodeExtension:fixup:codeLens:clicked', { op: 'accept', hasV2Event: true })
+                telemetryRecorder.recordEvent('cody.fixup.codeLens.accept', 'clicked', {
+                    privateMetadata: { op: 'accept' },
+                })
                 return this.accept(id)
             }),
             vscode.commands.registerCommand('cody.fixup.codelens.error', id => {
-                telemetryService.log('CodyVSCodeExtension:fixup:codeLens:clicked', { op: 'show_error' })
+                telemetryService.log('CodyVSCodeExtension:fixup:codeLens:clicked', {
+                    op: 'show_error',
+                    hasV2Event: true,
+                })
+                telemetryRecorder.recordEvent('cody.fixup.codeLens.error', 'error', {
```

Comment:
I wonder if a more appropriate translation for this particular event might be:

```suggestion
                telemetryService.log('CodyVSCodeExtension:fixup:codeLens:clicked', {
                    op: 'show_error',
                    hasV2Event: true,
                })
                telemetryRecorder.recordEvent('cody.fixup.codeLens.show_error', 'clicked', {
```